gateways/checkoutdotcom/request_builder.go

```diff
@@ -0,0 +1,59 @@
+package checkoutdotcom
+
+import (
+	"github.com/BoltApp/sleet"
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/payments"
+)
+
+func buildChargeParams(authRequest *sleet.AuthorizationRequest) *payments.Request {
+	var source = payments.CardSource{
+		Type: "card",
+		Number: authRequest.CreditCard.Number,
+		ExpiryMonth: uint64(authRequest.CreditCard.ExpirationMonth),
+		ExpiryYear: uint64(authRequest.CreditCard.ExpirationYear),
+		Name: authRequest.CreditCard.FirstName + " " + authRequest.CreditCard.LastName,
+		CVV: authRequest.CreditCard.CVV,
```

Comment:
again something present in the tokenizer, we will not get cvv in hail.

Reply:
Good catch- I double checked the checkout.com docs and cvv isn't a required field, so nil/empty should work. I'll make sure to verify when I get the test credentials. For now I'll still pass here to match other implementations.